src/components/bookmarks-panel/upload-panel-item.tsx

```diff
@@ -28,21 +28,26 @@ const Title = styled.div`
   color: ${({ theme }) => theme.colors.fontColor};
 `;
 
-const ButtonsContainer = styled.div`
+const ButtonsContainer = styled.div<{ justify: string }>`
   display: flex;
-  justify-content: space-between;
+  flex-direction: row;
+  justify-content: ${(props) => props.justify};
 `;
 
 type UploadPanelItemProps = {
   title?: string;
   children: React.ReactNode;
+  okButtonText?: string;
+  cancelButtonText?: string;
```

Comment:
I don't see any reason to have these props. ok is always "Next", cancel is always "Cancel". The component is not generic so there is no sense to add these properties.

Reply:
Done.